Automattic/woocommerce-payments
# Payment method checkbox mobile view

Fixes #2466

#### Overview
The issue describes that the checkboxes of payment methods look distorted in the mobile view. This issue is not reproducible anymore. However, in the issue, there is a design improvement suggestion for mobile view in [this comment](https://github.com/Automattic/woocommerce-payments/issues/2466#issuecomment-889318144). The mobile view of checkboxes for payment methods in this PR has been improved as per the design suggestion.

#### Changes proposed in this Pull Request
- Hiding the payment method icon in the mobile view. Targeting the Icon's SVG in the checkbox and hiding the display for mobile view.  
- Showing checkbox items are split into two lines. Making the first element of the flex container a full-width element to send the others in the second line if in mobile view.

The desktop view should be the same as before
![Desktop View](https://user-images.githubusercontent.com/33387139/130210938-5e122afc-3131-4a5f-974d-6981a47a4df1.png)

The mobile view would be changed as following:
![Mobile View](https://user-images.githubusercontent.com/33387139/130213771-3dbfece1-de73-4356-949a-408fcc4f1747.png)

#### Testing instructions

- Check `Enable UPE Checkout` in the `WCPay Dev Utils`.  Go to `WooCommerce -> Settings -> Payments -> WooCommerce Payments` and click `Add payment method`. The desktop view should be unchanged, the mobile view should reflect the changes mentioned above.
- Uncheck `Enable UPE Checkout` in the `WCPay Dev Utils`. Go to Onboarding Task in `WooCommerce -> Home`. Check the `Boost your sales with payment methods` section. The desktop view should be unchanged, the mobile view should reflect the changes mentioned above.

#### QA Checklist

- [ ] Desktop view is unchanged
- [ ] Mobile view does not show the payment method icon
- [ ] Mobile view has payment method name in one line and the other elements in the second line

-------------------

- [x] Changelog entry does not apply)
- [x] Test coverage is not needed
